src/lib/output.ts

```
@@ -1,10 +1,62 @@
+function toonEscapeValue(v: unknown): string {
+  if (v === null || v === undefined || (typeof v === 'number' && !isFinite(v))) return 'null';
+  if (typeof v === 'boolean') return v ? 'true' : 'false';
+  if (typeof v === 'number') return String(v);
+  const s = String(v);
+  const needsQuoting =
+    s === '' ||
+    s === 'true' || s === 'false' || s === 'null' ||
+    /^[+-]?[0-9]+(\.[0-9]+)?([eE][+-]?[0-9]+)?$/.test(s) ||
+    /^[ \t]|[ \t]$/.test(s) ||
+    /^[-#]/.test(s) ||
+    /[:,"\\{}\[\]]/.test(s) ||
```

ESLint

[error] Unnecessary escape character: \[.

(no-useless-escape)